sphinx/ext/autodoc/importer.py

```diff
@@ -84,13 +84,13 @@ def __getattr__(self, name):
 class _MockImporter(object):
     def __init__(self, names):
         # type: (List[str]) -> None
-        self.base_packages = set()  # type: Set[str]
-        for n in names:
+        self.base_packages = set()  # type: Set[Tuple[str, ...]]
```

Comment:
Is this base package? It is better to me to rename like `target_packages` this because the purpose is changed.

Reply:
Ok